src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -8,6 +8,7 @@
 
 :root {
 	--nav-sub-mobile-padding: 8px;
+	--search-typename-width: 6.75rem;
```

Comment:
Did you intentionally increase this from `6.25rem` to `6.75rem`?

Reply:
Yes because the whitespace is not displayed anymore because of `display: flex`. The rendering is the same though.